auth4genai/mcp/intro/why-auth-for-mcp.mdx

```diff
@@ -5,10 +5,10 @@ sidebarTitle: Why use Auth for MCP
 ---
 
 <Note>
-  Auth for MCP is currently available in Early Access and it's available for Universal Login flows only.
+Auth for MCP is currently available in Early Access. To join the Early Access program, please complete [this form](https://forms.gle/hvJ1ZRLmHr9YjV2a9). We'll reach out to you when your request is processed.
```

Comment:
Instead of adding the form here would it be better to link to [the early access section](https://auth0.com/ai/docs/mcp/intro/overview#early-access)?